test/verify/check-system-services

```diff
@@ -530,28 +530,33 @@ ExecStart=/bin/sh -c "while true; do sleep 5; done"
 
         # service a
         b.wait_js_cond('window.location.hash === "#/test-a.service"')
+        b.click("#service-details-show-relationships  button")
         b.wait_visible(rel_sel("Before", "test-b.service"))
         b.wait_visible(rel_sel("Conflicts", "test-c.service"))
         b.click(rel_sel("Before", "test-b.service"))
 
         # service b
         b.wait_js_cond('window.location.hash === "#/test-b.service"')
+        b.click("#service-details-show-relationships  button")
```

Comment:
Do these tests cover the trigger relationships somewhere, to make sure they work as intended?

Reply:
There are some relationships test, but I will check the trigger relationships explicitly in addition.

Reply:
there is already a test for it: see string:

self.assertIn(b.text("#Triggers .pf-c-description-list__text"), ["dbus.service", "dbus-broker.service"])